### PR TITLE
Update FOREX_USD oracle address

### PR DESCRIPTION
**Pull request by rellfy**

### CODE REVIEW

**Comment by rellfy** [comment]
The subgraph has finished syncing with the new FOREX oracle, it looks right.
You can try it [here](https://api.thegraph.com/subgraphs/name/handle-fi/handle/graphql) with
```graphql
query {
  chainlinkRate(id: "0xdb298285fe4c5410b05390ca80e8fbe9de1f259b") {
    value
  }
}
```
(it returns an 8 digit number in FOREX/USD)

**Comment by Preston-Harrison** [comment]
nice!

File: config/arbitrum.json [hunk 1]
```diff
@@ -7,7 +7,7 @@
     "rewardPool": "0xDE17Af0E4A6c870762508DcB7dCc20719584CBd0",
     "forex": "0xdb298285fe4c5410b05390ca80e8fbe9de1f259b",
     "fxTransformer": "0x6a67CB66B928629DE164d7CE55fB1Bd77F31e553",
-    "FOREX_USD": "0x89c375CAff8176085C13E1603f05f482C6F939e2",
+    "FOREX_USD": "0xe12e73c06f983ffd535840033973a1b382fc8df8",
```
Comment (bitcoinbrisbane) [inline config/arbitrum.json:10]: is this supposed to be the token or the aggregator?  https://arbiscan.io/address/0xe12e73c06f983ffd535840033973a1b382fc8df8#code

**Review by bitcoinbrisbane — COMMENTED** [review]
Just trying to understand oracle vs aggregator

File: config/arbitrum.json (hunk 1 above)
Comment (rellfy) [reply config/arbitrum.json:10]: It is the aggregator

**Comment by rellfy** [comment]
@nabsta-eth @Preston-Harrison @bitcoinbrisbane please review

**Review by Preston-Harrison — APPROVED** [review]

**Review by nabsta-eth — APPROVED** [review]

**Pull request merged**